index.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
```

Comment:
Przydałoby się zrezygnować z tych odstępów:
![image](https://github.com/devmentor-pl/task-html-and-css-basics/assets/55095257/b5240e5c-1840-4101-9e5b-a5443c239454)

To winna `margin` dla `row`. Może lepszy będzie sam `padding`?